include/enrico/surrogate_heat_driver.h

```diff
@@ -13,6 +13,144 @@
 
 namespace enrico {
 
+//! Class containing geometric information for a flow channel
+class Channel {
+  public:
+    Channel() = default;
+
+    //! Channel index
+    int index_;
+
+    //! Channel flow area
+    double area_;
+
+    //! Vector of rod IDs connected to this channel, all with a fractional perimeter
+    //! in contact with the channel equal to 0.25
+    std::vector<int> rod_ids_;
+};
+
+//! Class containing geometry information for a cylindrical solid rod
+class Rod {
+  public:
+    Rod() = default;
+
+    //! Rod index
+    int index_;
+
+    //! Rod cladding outer radius
+    double clad_outer_radius_;
+
+    //! Rod cladding inner radius
+    double clad_inner_radius_;
+
+    //! Rod pellet radius
+    double pellet_radius_;
+
+    //! Vector of channel IDs connected to this rod, all with a fractional perimeter
+    //! in contact with the rod equal to 0.25
+    std::vector<int> channel_ids_;
+};
```

Comment:
`Channel` and `Rod` basically look like structs. You might want to consider just marking them as such.

Reply:
Yeah, good point